talk/owt/sdk/base/windowcapturer.cc

```diff
@@ -63,30 +63,31 @@ void BasicWindowCapturer::WindowCaptureThreadFunc(void* pThis) {
 }
 
 bool BasicWindowCapturer::CaptureThreadProcess() {
-  if (!capture_started_) {
+  while (!capture_started_) {
```

Comment:
Retry limitation for the loop?

Reply:
Will you add a limitation here or it's ready to merge?

Reply:
adding a limit will cause issue as we don't know when user will finally confirm the window to be shared. the thread will be destroyed.